share/errmsg-utf8.txt

```diff
@@ -18724,6 +18724,15 @@ ER_BINLOG_CANT_READ_INDEX
 ER_RPL_FAILED_IN_RLI_INIT_INFO
   eng "Failed in %s called from Relay_log_info::rli_init_info()."
 
+ER_INVALID_UUID
```

Comment:
These error codes should be mapped to the 5.6 section. I think this replaces ER_PLACEHOLDER_50030, with ER_UUID_NOT_IN_EXECUTED_GTID_SET replacing ER_PLACEHOLDER_50032.